core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -58,7 +58,7 @@ THE SOFTWARE.
           permission="${it.ARTIFACTS}" />
 
         <j:set var="set" value="${it.changeSet}" />
-        <t:summary icon="symbol-list">
+        <t:summary icon="symbol-changes">
```

Comment:
That doesn't look better either imo, not every change is caused by a PR.
What do you or others think about using `document-text`, `documents` or the already added details one?

Reply:
Hmm might be a little too similar to the newer 'Status' icon, unless there's a better icon for that too? - 

<img width="340" alt="image" src="https://user-images.githubusercontent.com/43062514/156944230-c2a4fb20-5c03-4ae9-8fe6-fed9e179a63f.png">

Reply:
I was referring to this one:
![](https://i.imgur.com/X2mAAfu.png)

Maybe try that one fits all occasions?
![](https://i.imgur.com/hWOC0RG.png)

Reply:
Looks good to me! Done :)